src/styles.css

```diff
@@ -0,0 +1,1198 @@
+  display: flex;
+  align-items: center;
+}
+
+.short_desc * {
+  margin-right: 10px;
+}
+
+.short_desc h4 {
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.short_desc h4:nth-child(1) {
+  color: #344563;
+  text-transform: uppercase;
+}
+
+.short_desc span {
+  background-color: #c1c7d0;
+  width: 10px;
+  height: 10px;
+  border-radius: 50%;
+}
+
+.blur {
+  filter: blur(5px);
+}
+
+@media screen and (min-width: 900px) {
+  .body {
+    margin: 0;
+    background-color: #e5e5e5;
+  }
+
+  .header-container {
+    display: none;
+  }
+
+  .web-nav {
+    display: flex;
+  }
+
+  .Mylogo {
+    text-decoration: none;
+    color: #6070ff;
+  }
+
+  .modal-example {
+    width: 60%;
+    padding: 2%;
+  }
+
+  .modal-example.show {
+    right: 0;
+    max-width: 100%;
+    bottom: 0;
+    max-height: 608px;
+    transform: translate(-10%, 0%);
+  }
+
+  .modal__body {
+    overflow-y: auto;
+  }
+
+  .modal-img {
+    width: 90%;
+  }
+
+  .desc {
+    width: 50%;
+  }
+
+  nav {
+    display: flex;
+    height: 80px;
+    width: 97%;
+    background: #fff;
+    align-items: center;
+    justify-content: space-between;
+    padding: 13px;
+    flex-wrap: wrap;
+  }
+
+  nav .logo {
+    color: #6070ff;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    transition: all 1s;
+  }
+
+  .logo:hover {
+    transform: rotate(360deg);
+  }
+
+  nav ul {
+    display: flex;
+    flex-wrap: wrap;
+    list-style: none;
+  }
+
+  nav ul li {
+    margin: 0 5px;
+  }
+
+  nav ul li a {
+    color: #344563;
+    text-decoration: none;
+    padding: 8px 15px;
+    border-radius: 5px;
+    letter-spacing: 1px;
+    transition: all 0.3s ease;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 20px;
+  }
+
+  nav ul li a:hover {
+    color: #fff;
+    background: #344563;
+  }
+
+  .content {
+    background-image: url(../assets/Header-bg.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+    width: 99%;
+    height: 100vh;
+    border-radius: 0 0 0 100px;
+    border-bottom: 1px solid #cfd8dc;
+    background-color: #fff;
+  }
+
+  .content p {
+    width: 63%;
+    margin-left: 20%;
+    margin-top: -5%;
+    line-height: 28px;
+    font-size: 20px;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .content h3 {
+    margin-left: 20%;
+    margin-top: -5%;
+  }
+
+  .content ul {
+    margin-left: 20%;
+    margin-top: 1%;
+  }
+
+  .intro {
+    width: 400px;
+    font-size: 48px;
+    line-height: 60px;
+    margin-left: 22%;
+    animation-name: introin;
+    animation-duration: 4s;
+  }
+
+  @keyframes introin {
+    from {
+      margin-left: 100%;
+    }
+
+    to {
+      margin-left: 22%;
+    }
+  }
+
+  .projects {
+    display: flex;
+    flex-direction: row;
+    margin: 9%;
+    width: 66%;
+    height: 60%;
```

Comment:
- [OPTIONAL] I  highly suggest you fix the cards' layout and smaller this part's font. Right now, projects are not very responsive and not centered:
![image](https://user-images.githubusercontent.com/91486791/236646070-d22819fb-88de-4416-bc3f-19a58f819c95.png)
![image](https://user-images.githubusercontent.com/91486791/236646123-aea9add8-416e-4edf-b0ed-655ad3e1901d.png)